go/kbfs/libfs/fs.go

```diff
@@ -124,7 +124,7 @@ func pathForLogging(
 		childName := n.ChildName(p)
 		ret = path.Join(ret, childName.String())
 		nextNode, _, err := config.KBFSOps().Lookup(ctx, n, childName)
-		if err != nil {
+		if err != nil || nextNode == nil {
```

Comment:
In the very rare case of an entry that gets a duplicated obfuscated name within a directory, this could give the wrong answer. in some cases.  So the "right" answer would be to try to follow the actual symlink here -- but it's probably not worth it.  Maybe just a comment note here about the symlink behavior so we don't get confused by it later?